internal/pprof/stub_test.go

```diff
@@ -0,0 +1,51 @@
+package pprof
+
+import (
+	"go/types"
+	"strings"
+	"testing"
+)
+import "golang.org/x/tools/go/packages"
```

Comment:
This is the only dependency of the entire module. Please consider moving the test to a sub-module, e.g `github.com/pyroscope-io/godeltaprof/compat`